### PR TITLE
chore: normalize packages version

### PR DESCRIPTION
**Pull request by carlos-r-l-rodrigues**

### CODE REVIEW

**Comment by carlos-r-l-rodrigues** [comment]
@olivermrbl has something changed to generate snapshots? I can't get that working.

**Comment by olivermrbl** [comment]
> @olivermrbl has something changed to generate snapshots? I can't get that working.

Hmm no, not that I know of. [The error messages are a bit cryptic](https://github.com/medusajs/medusa/actions/runs/10290389893). I can have a look tomorrow.

Just tried to re-run it.

File: .changeset/beige-readers-pull.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
```
Comment (olivermrbl) [inline .changeset/beige-readers-pull.md]: should be removed

**Review by olivermrbl — APPROVED** [review]
LGTM, minor thing

File: .changeset/beige-readers-pull.md (hunk 1 above)
Comment (carlos-r-l-rodrigues) [reply .changeset/beige-readers-pull.md]: removed it.
I've added it to see if that was the issue to generate a snaptshot :/

**Pull request merged**